app/domain/authentication/security/extract_request_remote_addr_info.rb

```diff
@@ -0,0 +1,51 @@
+require 'command_class'
+
+module Authentication
+  module Security
+    ExtractRequestRemoteAddrInfo = CommandClass.new(
```

Comment:
where will this class be used? 

I am not sure that this needs to be a command class, but rather you can take the only line in this call method `extract_request_remote_addr_info` and just implement it in the calling class

Reply:
Another `CommandClass` `ValidateProxy` will use it.

Reply:
i'll approve this PR to not block it but once we create the PR that consumes this class i would like us to challenge its existence. We may be better off implementing this as a method in `ValidateProxy`

Reply:
I think the name of the command class is confusing because the `remote_addr` is the IP in the TCP packet,
but here we are performing logic both on the XFF and the remote_addr to return a client ip and a proxy list. which should be validated later